docs/cloud/faq/cluster_provisioning.md

```diff
@@ -0,0 +1,10 @@
+# Cluster Provisioning
+
+### Are there plans to support automatic VM resize? Either online or offline, but without having to go to backups.
+
+You will be able to do an online vm resize but it wont be auto scaling. 
+You will have to initiate it yourself
+
+### Are there plans to support automatic Disk Resize resize? Either online or offline, but without having to go to backups
+
+No, you have to initiate it yourself
```

Comment:
**QUESTION:** What does `initiate` mean in this context? It may be good to be more explicit. What steps would be needed? Contacting support? Creating a new cluster with bigger disk storage and doing restore?

Reply:
Do we have an option that users can scale out their VM? I thought that's not yet available.

Reply:
no, it's all through backup - restore ATM

Reply:
I fixed some of the working, hopefully it's clearer now

Reply:
It's in my PR